src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -21,24 +22,32 @@ BmxConfig config
 		bool nonInteractive,
 		bool ignoreCache
 	) {
+		var orgSource = ParameterSource.CliArg;
+		if( string.IsNullOrEmpty( org ) && !string.IsNullOrEmpty( config.Org ) ) {
```

Comment:
aside: writing all this overwrite/print/prompt logic really has a fizz buzz feel - it feels like you should be able to consolidate some actions and/or logical branches, but actually you can't...